solr/solrj/src/java/org/apache/solr/common/params/CollectionParams.java

```diff
@@ -70,6 +70,19 @@ public boolean isHigherOrEqual(LockLevel that) {
     }
   }
 
+  /**
+   * <p>(Mostly) Collection API actions that can be sent by nodes to the Overseer over the <code>/overseer/collection-queue-work</code>
+   * ZooKeeper queue.</p>
+   *
+   * <p>Some of these actions are also used over the cluster state update queue at <code>/overseer/queue</code>, and really
```

Comment:
Minor: link to Overseer.ClusterStateUpdater instead of just naming the queue here

Reply:
Now that I try to change it, I remember I tried initially but there's no class visibility from here to Overseer. Sometimes I wish Javadocs were more permissive we are not linkers, we are humans.